Simplenote/src/test/java/com/automattic/simplenote/repositories/FakeTagsRepository.kt

```diff
@@ -0,0 +1,35 @@
+package com.automattic.simplenote.repositories
+
+import com.automattic.simplenote.models.Tag
+import com.automattic.simplenote.utils.TagUtils
+
+class FakeTagsRepository : TagsRepository {
```

Comment:
I think we should rather use a `mock` than a `fake`. This class contains a lot of logic and makes the test a little bit error prone. Let me know what you think 👍

Reply:
Yeah, the fake is harder to maintain. Fixed in 7ca591ddf390341f3153a9cd791eca16c7a4925c.